SampleApplications/Workshop/Aggregation/ConsoleAggregationServer/Quickstarts.AggregationServer.Config.xml

```diff
@@ -55,7 +55,7 @@
   </TransportQuotas>
   <ServerConfiguration>
     <BaseAddresses>
-      <ua:String>opc.tcp://localhost:62541/AggregationServer</ua:String>
+      <ua:String>opc.tcp://localhost:4840/AggregationServer</ua:String>
```

Comment:
Is this a requirement? It can cause a conflict running the server locally when LDS is already installed, so maybe a second config for docker with this port is an option..

Reply:
It is not necessary. It is simply the port most folks will remember as it is the default UA port. I thought about the conflict with LDS, but most folks don't use it anyway. I'm indifferent, though. Want me to change it back?

Reply:
Hi Erich, I *think* its ok to repurpose the UA Aggregation solution to VS2019 and update Readme in the project folder. The old server port is the same as the Reference server one, so it wasn't a good choice anyway. But 4840 doesn't start for me and maybe others, please pick a free one, e.g 62530. Then the aggregation server can be tested with all Quickstart servers locally. Appveyor build needs to be fixed, I can check that..